Unix/installbuilder/GNUmakefile

```diff
@@ -36,7 +36,11 @@ endif
 
 # New output format for OMI on Linux (avoids ugly renaming since we always build universal)
 ifeq ($(PF),Linux)
-OUTPUTFILE_LINE = --OUTPUTFILE=omi-$(CONFIG_VERSION)-$(CONFIG_PATCH_LEVEL).universal.$(PF_ARCH)
+  ifneq ($(PF_ARCH),ppc)
```

Comment:
Positive logic is easier to follow. `ifeq ($(PF_ARCH),ppc)` and switch the statements.